docs/codeql/reusables/supported-versions-compilers.rst

```diff
@@ -24,6 +24,7 @@
    JavaScript,ECMAScript 2022 or lower,Not applicable,"``.js``, ``.jsx``, ``.mjs``, ``.es``, ``.es6``, ``.htm``, ``.html``, ``.xhtm``, ``.xhtml``, ``.vue``, ``.hbs``, ``.ejs``, ``.njk``, ``.json``, ``.yaml``, ``.yml``, ``.raml``, ``.xml`` [7]_"
    Python [8]_,"2.7, 3.5, 3.6, 3.7, 3.8, 3.9, 3.10, 3.11",Not applicable,``.py``
    Ruby [9]_,"up to 3.2",Not applicable,"``.rb``, ``.erb``, ``.gemspec``, ``Gemfile``"
+   Swift,"Swift 5.4-5.7","Swift compiler","``.swift``"
```

Comment:
I'd say Swift needs a footnote explaining it's in beta, and maybe also specifying that
* Windows is not supported
* Linux support is partial (only works with Swift 5.7.3 at the moment)

Reply:
Good point.  Indeed, Kotlin already has a note saying "Kotlin support is currently in beta.", so we probably want that _and_ the details you've given above.

Reply:
I've added a footnote, please could you check you're happy with it.

Reply:
perfect :+1: